src/providers/commandsProvider.ts

```diff
@@ -94,9 +94,6 @@ function execCommand(currentRedPath: string, command: string, fileUri?: vscode.U
         filePath = fileUri.fsPath;
     }
 
-    if (filePath.indexOf(' ') > 0) {
```

Comment:
Could have simply removed the `if` condition here. Altho this works about the same